src/cli/backup/teams.go

```diff
@@ -0,0 +1,190 @@
+package backup
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+
+	"github.com/alcionai/corso/src/cli/flags"
+	"github.com/alcionai/corso/src/cli/utils"
+	"github.com/alcionai/corso/src/pkg/path"
+)
+
+// ------------------------------------------------------------------------------------------------
+// setup and globals
+// ------------------------------------------------------------------------------------------------
+
+const (
+	teamsServiceCommand                 = "team"
```

Comment:
`Teams`, with the s, is the formal product name.
```suggestion
	teamsServiceCommand                 = "teams"
```